### DISCUSSION TITLE
can I get a css file for each route?

### DISCUSSION THREAD

**Discussion by lili21** [chakra-ui/panda#881] [Q&A]
I'm using panda with the Remix framework. 
Right now, all style will be generated to one single css file, which not great for performance.

**Comment by astahmer** [comment]
kind-of ? we have the `chunks` folder in the generated `outdir` (defaults to `styled-system`) that contains the CSS generated for each files when running the `panda` command (or `panda cssgen`) but it was meant as a debugging helper rather than a production ready CSS (afaik it doesn't get minified or hashed)

I'm not sure how we can achieve this with a smooth DX tho, cause if we just kept those chunks as-is, you'd have to import them one by one in each of your files which.. doesn't seem ideal ? do you have a better solution in mind ?
